lib/internal/fs.js

```diff
@@ -18,16 +19,16 @@ const {
 
 function assertEncoding(encoding) {
   if (encoding && !Buffer.isEncoding(encoding)) {
-    throw new Error(`Unknown encoding: ${encoding}`);
+    throw new errors.TypeError('ERR_INVALID_OPT_VALUE', 'encoding', encoding);
```

Comment:
Definitely prefer a more specific error for `Unknown encoding`. We throw these in several places throughout the source, often enough that it makes sense to identify them separately.